pyproject.toml

```diff
@@ -98,3 +98,18 @@ all = [
 [tool.isort]
 profile = "black"
 known_third_party = ["fastapi", "pydantic", "starlette"]
+
+[tool.pytest.ini_options]
+addopts = [
+  "--strict-config",
+  "--strict-markers",
+]
+xfail_strict = true
+junit_family = "xunit2"
```

Comment:
I was just going to ask about each of these options but after a bit of research now I understand the reason for them.

So, in short, the strict configs (as you describe it in the title) and the future version of files with the `junit_family`. Right?

Reply:
yep, I think pytest 7 will make `--strict` an alias to `--strict-config --strict-markers` but currently `--strict` is a deprecated alias to `--strict-markers`